node/pkg/query/query_test.go

```diff
@@ -36,6 +36,7 @@ const (
 	// Speed things up for testing purposes.
 	requestTimeoutForTest = 100 * time.Millisecond
 	retryIntervalForTest  = 10 * time.Millisecond
+	auditIntervalForTest  = 10 * time.Millisecond
```

Comment:
Did you try a test scenario where the audit and retry intervals are different just in case there is some weird interaction?

Reply:
No, I didn't, but this behavior is exactly the same as before.